### PR TITLE
schema interop additions

### PR DESCRIPTION
**Pull request by jweissm**

### CODE REVIEW

**Comment by clenk** [comment]
Tests will continue to fail until `ajv-cli` resolves this issue: https://github.com/ajv-validator/ajv-cli/issues/172

**Review by clenk — CHANGES_REQUESTED** [review]
1. These 2 also need to switch to draft/2020-12:
- schemas/common/core.json
- schemas/sros/sighting.json

2. In `test_examples.sh`, `ajv` needs to use `--spec=draft2020`. I think we can remove the `-c ajv-formats-draft2019` option.

File: interop/intrusion-set.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "allOf": [
+        {
+          "$ref": "core.json"
+        },
+        {
+           "$ref": "../schemas/sdos/intrusion-set.json"
+        }
+    ],
+    "required": [
+        "resource_leve",
```
Comment (clenk) [inline interop/intrusion-set.json:12]: Typo!
```suggestion
        "resource_level",
```

**Comment by clenk** [comment]
This looks good to me.

**Comment by ejratl** [comment]
Merging this will require an update to the workflows as well, specifically
https://github.com/oasis-open/cti-stix2-json-schemas/blob/master/.github/workflows/python-ci-tests.yml#L28
installs ajv-formats-draft2019. The test-example.sh shows errors but does not error out, so the check passes but with errors.

**Comment by ejratl** [comment]
After the change in PR#154, it still isn't entirely happy since the schemas already existed.

**Pull request merged**